cmd/infracost/comment_gitlab.go

```diff
@@ -115,7 +115,7 @@ func commentGitLabCmd(ctx *config.RunContext) *cobra.Command {
 				if posted {
 					cmd.Println("Comment posted to GitLab")
 				} else {
-					cmd.Println("Comment not posted to GitHub (GitLab)")
+					cmd.Println("Comment not posted to GitLab (skipped)")
```

Comment:
Haha I noticed this too in https://github.com/infracost/infracost/pull/2587. We will get this one merged first and then I'll fix up conflicts.